src/client/src/rt-platforms/browser/browser.ts

```diff
@@ -39,9 +40,7 @@ export default class Browser implements Platform {
     open: (config: WindowConfig, onClose?: () => void) => openBrowserWindow(config, onClose),
   }
 
-  fdc3 = {
-    broadcast: () => {},
-  }
+  intents = new Noop()
```

Comment:
Is there a way we can do this so that platforms don't have to explicitly set intents to `Noop`? Just so they can opt-in as opposed to having to opt out...

Reply:
I guess we could have another wrapper/abstraction around the methods which check they exist. I mainly did this for typing purposes. I'll leave as is here, but let's iterate on the other branch.